docs/concepts/jcloud/index.md

```diff
@@ -164,12 +196,14 @@ To access the [Grafana](https://grafana.com/)-powered dashboard, first get {ref}
 :width: 80%
 ```
 
-### List Flows
+### List Flows, Secrets or Jobs
+
+#### List Flows
 
-To list all of your "Serving" Flows:
+To list all of your "Starting", "Serving", "Failed", "Updating", and "Paused" Flows:
 
 ```bash
-jc list
+jc list flows
```

Comment:
Is this a breaking change? Does `jc list/update/remove` etc need params `flow/job/secret` etc now?

Reply:
Yes we need this to distinguish between the resource to create.

Reply:
but can we have `flow` as default? to avoid breaking changes?